app/src/routes/ChoiceBoard.tsx

```diff
@@ -20,10 +22,59 @@ function ChoiceBoard() {
   // eslint-disable-next-line
   const { selectedItems, appendSelection } = useChoiceBoardModel();
 
+  const [dismissed, setDismissed] = useState(false);
+  const animationDuration = 750;
+
+  // key variables for refreshing cards
+  const [key1, setKey1] = useState(`${new Date().getTime()}${1}`);
+  const [key2, setKey2] = useState(`${new Date().getTime()}${2}`);
+
+  useEffect(() => {
+    const refreshCards = () => {
+      setDismissed(false);
+
+      // After dismissing, get new data and re-render the cards here:
+      setKey1(`${new Date().getTime()}${1}`);
+      setKey2(`${new Date().getTime()}${2}`);
+    };
+
+    const timer = setTimeout(() => {
+      if (dismissed) refreshCards();
+    }, animationDuration * 2);
+    const cleanup = () => clearTimeout(timer);
+    return cleanup;
+  }, [dismissed]);
+
   const commonOnClick = (_, card: never) => {
     appendSelection(card);
   };
 
+  const dismissButtonLabel = "I don't like either 🤔. Next, please!";
+
+  const handleDismiss = (event: MouseEvent<HTMLButtonElement>): void => {
+    event.preventDefault();
+
+    const card1 = document.getElementById('card1');
+    const card2 = document.getElementById('card2');
+
+    anime({
+      targets: card1,
+      translateX: -1500,
+      duration: animationDuration,
+      rotate: '180deg',
+      easing: 'easeInElastic(1, 1)',
+    });
+    anime({
+      targets: card2,
+      translateX: +1500,
+      duration: animationDuration,
+      rotate: '180deg',
+      easing: 'easeInElastic(1, 1)',
+    });
+
+    setDismissed(true);
+  };
+
```

Comment:
Right now the animation looks good but when we click on the dismiss button a horizontal scroll bar is visible for few seconds. Please make the animation more subtle something like [this](https://animista.net/play/exits/slide-out/slide-out-right)